lib/pages/chat/chat.dart

```diff
@@ -375,7 +378,9 @@ class ChatController extends State<Chat>
         setReadMarker();
         return;
       }
-      _initUnreadLocation(fullyRead);
+      if (room?.hasNewMessages == true) {
+        _initUnreadLocation(fullyRead);
+      }
```

Comment:
why we need this change ?

Reply:
If don't have a new message, always call `_initUnreadLocation`  and jump to the last read message. 
So we need to check if hasNewMessgae => call `_initUnreadLocation `